### DISCUSSION TITLE
Centering header text doesn't work

### DISCUSSION THREAD

**Discussion by ChimpGamer** [Power-Components/livewire-powergrid#1760] [BUG]
In the documentation I saw that it is possible to add attributes to the header and the body. So I've tried the example in my project but for the header the text-center attribute doesn't work.
This is the part of the documentation I'm referring to: https://livewire-powergrid.com/table-component/component-columns.html#headerattribute
![image](https://github.com/user-attachments/assets/c88f8b8f-a990-46d7-b618-8ccaa95b09b6)
![image](https://github.com/user-attachments/assets/7ae3beb0-5b61-4606-a539-aab443ee46d5)

**Comment by MrYamous** [comment]
IIUC the header text is in a flex div, not directly under th element concerned by the class you defined and this is preventing this style to be applied as expected
Maybe this can be done using a custom theme

**Reply by ChimpGamer** [reply]
It probably can be done using a custom theme but it should be possible to just do it per column like the example in the documentation.

**Comment by ChimpGamer** [comment]
This is still not fixed with Tailwind v4. Anyone that could assist me with this?
